storage/container_storage_interface/persistent-storage-csi-azure.adoc

```diff
@@ -31,6 +31,17 @@ After full migration, in-tree plugins will eventually be removed in later versio
 
 include::modules/persistent-storage-csi-azure-disk-sc-zrs.adoc[leveloffset=+1]
 
+ifndef::openshift-rosa,openshift-dedicated[]
+include::modules/persistent-storage-byok.adoc[leveloffset=+1]
+
+[NOTE]
+====
+If you do not set up user-managed encryption, but the OS (root) disk is encrypted, the storage class uses the OS disk encryption key by default  to encrypt provisioned storage volumes.
```

Comment:
@lpettyjo Suggest to reword to:
If the OS (root) disk is encrypted and there is no encrypted key defined in storage class, Azure-Disk CSI Driver uses the OS disk encryption key by default to encrypt provisioned storage volumes.

Reply:
Done. Thanks!